ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -1,19 +1,22 @@
-from typing import Any, Callable, Tuple
+from typing import Any, Callable, Tuple, Union
 
 import torch
 
 from ignite.metrics import EpochMetric
 
 
-def precision_recall_curve_compute_fn(y_preds: torch.Tensor, y_targets: torch.Tensor) -> Tuple[Any, Any, Any]:
-    try:
-        from sklearn.metrics import precision_recall_curve
-    except ImportError:
-        raise RuntimeError("This contrib module requires sklearn to be installed.")
+def precision_recall_curve_compute_fn(y_preds: torch.Tensor, y_targets: torch.Tensor) -> Union[Any, Any, Any]:
+    from sklearn.metrics import precision_recall_curve
 
-    y_true = y_targets.numpy()
-    y_pred = y_preds.numpy()
-    return precision_recall_curve(y_true, y_pred)
+    y_true = y_targets.cpu().numpy()
+    y_pred = y_preds.cpu().numpy()
+    precision, recall, thresholds = precision_recall_curve(y_true, y_pred)
```

Comment:
I think mypy shouldn't be happy with variable redefinition with different types...
```suggestion
    np_precision, np_recall, np_thresholds = precision_recall_curve(y_true, y_pred)
```